packages/core/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { Boundary } from "../../common/boundary";
+import * as Classes from "../../common/classes";
+import { Position } from "../../common/position";
+import { IProps } from "../../common/props";
+import { Menu } from "../menu/menu";
+import { MenuItem } from "../menu/menuItem";
+import { IOverflowListProps, OverflowList } from "../overflow-list/overflowList";
+import { IPopoverProps, Popover } from "../popover/popover";
+import { Breadcrumb, IBreadcrumbProps } from "./breadcrumb";
+
+export interface IBreadcrumbsProps extends IProps {
+    /**
+     * Callback invoked to render visible breadcrumbs. If
+     * `currentBreadcrumbRenderer` is also supplied, that callback will be used
+     * for the current breadcrumb instead.
+     *
+     * If this callback is not supplied, a `Breadcrumb` will be rendered.
```

Comment:
proposal?
```suggestion
     * @default Breadcrumb
```
followup: does it work to make the default value just the component reference?

Reply:
It would be, but only because it's an SFC. I'd rather not rely on that implementation detail. It's good enough for docs though.